docs/docs/streamr-network/incentives/network-incentives.md

```diff
@@ -30,6 +30,109 @@ The introduction of token economics defines three new roles: [Sponsor](../networ
+
+The ERC-20 "delegation token" works in a manner similar to a liquidity pool: during delegation, DATA tokens are swapped for newly minted delegation tokens; and undelegation again returns the DATA tokens and burns the corresponding amount of delegation tokens. This exchange rate set by the [`DefaultExchangeRatePolicy`](https://github.com/streamr-dev/network-contracts/blob/master/packages/network-contracts/contracts/OperatorTokenomics/OperatorPolicies/DefaultExchangeRatePolicy.sol) is such that the total supply of operator tokens corresponds to the total amount of DATA tokens either held in the contract or staked in Sponsorship contracts (the `valueWithoutEarnings` of the Operator contract). In the code, it's called "Operator token", and for the most part its existence is hidden from the user interface, APIs and the subgraph schema.
+
+Being an ERC-20, the Operator contract address can be added to wallet and its delegation tokens transferred, albeit with two restrictions: one for the operator and one for the rest of the delegators. The operator's self-delegation must remain above the [`minimumSelfDelegationFraction`](https://polygonscan.com/address/0x869e88dB146ECAF20dDf199a12684cD80c263c8f#readProxyContract) (currently set to 5%) of the total "delegation token" supply. Other delegations must still be above [`minimumDelegationWei`](https://polygonscan.com/address/0x869e88dB146ECAF20dDf199a12684cD80c263c8f#readProxyContract) (currently set to 1 delegation token), or completely undelegate to zero. The value of 1 delegation token starts at 1 DATA, but appreciates as profits accumulate. This limitation prevents rounding error shenanigans in contracts with very low self-delegation.
+
+The outcome of the linear exchange rate between DATA and delegation token is: in the user interfaces we can ignore the delegation token and just convert the values to DATA by simple multiplication. The profits are shared in direct proportion to delegations, much like in Sponsorships where stakers share the sponsorship in direct proportion to their stake. Should a non-linear bonding curve be introduced later, the interpretation should remain the same: the value of the delegation tokens is the amount of DATA tokens the delegator would receive after undelegating them. Note that the DATA tokens themselves might not exist in the contract yet, since mostly we expect the Operator contract's value be staked out into Sponsorship contracts. But delegation bookkeeping is still done in the form of the ERC-20 delegation token.
```

Comment:
Maybe simpler if we don't talk about a possible future functionality

```suggestion
The outcome of the linear exchange rate between DATA and delegation token is: in the user interfaces we can ignore the delegation token and just convert the values to DATA by simple multiplication. The profits are shared in direct proportion to delegations, much like in Sponsorships where stakers share the sponsorship in direct proportion to their stake. Note that the DATA tokens themselves might not exist in the contract yet, since mostly we expect the Operator contract's value be staked out into Sponsorship contracts. But delegation bookkeeping is still done in the form of the ERC-20 delegation token.
```